news/2018-07-22-osu-mapping-olympiad-3-mania.md

```diff
@@ -0,0 +1,57 @@
+
+For those who are unaware, the osu! Mapping Olympiad is a revival of official beatmapping contests, featuring a wide array of competitions ranging from traditional spreads to more unique contests to speedmapping challenges and more!
+
+To kick off osu!mania's first round of the Mapping Olympiad, we'll be adhering to the more traditional and well known style of competition we're all familiar with, but there will be more opportunities for interesting and unique takes in further iterations.
+ 
+## Song
+To kick off the contest, we've picked a song we believe will provide a concrete foundation for a variety of styles, made by one of our featured artists, **Dictate**!
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/21/previews/420.mp3" type="audio/mpeg">
+</audio>
+
+Dictate's *Scavenger* is a song with a very unorthodox structure. This masterpiece of oddly syncopated synths and gritty sound is sure to give rise to many experimental interpretations from those expanding beyond the mapping meta.
+ 
+## Rules
+- This contest is for **osu!mania** mode with all rankable key modes (4K-9K) only.
+- Each entry must follow the **[Ranking Criteria](https://osu.ppy.sh/help/wiki/Ranking_Criteria)**. Entries that break the Ranking Criteria will be subject to penalties ranging from point deductions to disqualification based on severity.
+- Beatmap submissions may **not** exceed 15MB.
+- Storyboards will **not** be considered in the judging process.
+- Custom hitsounds are allowed.
+- Your submission must contain a full spread of 3 to 5 difficulties of the same key mode with Normal being the lowest difficulty. We will not strictly enforce the <2.00 star rating limit for the lowest difficulty.
+- Collaboration on entries is **not** permitted.
+- Maps submitted to the contest must not be uploaded with the Beatmap Submission System (BSS) until the contest is concluded and the winners announced.
+ 
+## Judging Criteria
+- **Musical Relevance (10 pts)**: How well the entry follows the outline of the song through rhythm choices, emphasis, and patterns.
+- **Creativity (10 pts)**: How unique and interesting the concepts and themes used are.
+- **Technique (10 pts)**: How well executed the ideas present in the entry are.
+- **Impression (10 pts)**: The judge's overall impression. How they view the effectiveness of the entry in its entirety.
+- **Hitsounding (5 pts)**: How well the entry utilizes hitsounds to add to overall chart expression.
+- **Difficulty Spread (5 pts)**: How well concepts are introduced in lower difficulties and developed in higher difficulties.
+
+**We are looking for capable judges! If you are interested, please make a post in the [Judging Call Thread](https://osu.ppy.sh/community/forums/posts/6737869) or send a message to [Protastic101](https://osu.ppy.sh/forum/ucp.php?i=pm&mode=compose&u=6712747). Please be aware that we can only accept people who have the resources (knowledge, time, reliability) to properly judge the entries.**
```

Comment:
fwiw, I don't have access to the Judging Call Thread

Reply:
Yeah it's in a private forum, will be moved to Beatmap Management when the newspost/contest goes live